products/graalvm-ce.md

```diff
@@ -21,9 +21,15 @@ auto:
 # Release dates can be found on https://www.graalvm.org/release-calendar/.
 # eol(x) = releaseDate(x+1).
 releases:
+  - releaseCycle: "25"
+    releaseDate: 2025-09-16
+    eol: false
+    latest: "25.0.0"
```

Comment:
```suggestion
    latest: "25"
```
are you sure that version as 25.0.0  ? becasuse in here it shown as first version( released one ) as 25 not 25.0.0 : https://www.graalvm.org/release-calendar/

Reply:
Hi @usta , indeed, I asked myself the same question 🤔 
<img width="893" height="509" alt="image" src="https://github.com/user-attachments/assets/a2ebaa33-5bc1-46a9-93a8-03c01651522c" />

I see that you used "just" `25` on https://github.com/endoflife-date/endoflife.date/pull/8469

BTW, I was not sure it was possible to get a latest version that has no minor semantic version 👍